docs/content/connectors/mongodb-cdc.md

```diff
@@ -226,6 +226,13 @@ Connector Options
       <td>Integer</td>
       <td>The max size of the queue to use when copying data.</td>
     </tr>
+    <tr>
+      <td>batch.size</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">0</td>
+      <td>Integer</td>
+      <td> Change stream cursor batch size. Specifies the maximum number of change events to return in each batch of the response from the MongoDB cluster. The default is 0 meaning it uses the servers default.</td>
```

Comment:
The default is 0 meaning it uses the servers default -> The default is 0 meaning it uses the server's default value.

Reply:
Done.